tests/unit/models/new-platform-flexberry-g-i-s-map-test.js

```diff
@@ -3,8 +3,15 @@ import { moduleForModel, test } from 'ember-qunit';
 moduleForModel('new-platform-flexberry-g-i-s-map', 'Unit | Model | new-platform-flexberry-g-i-s-map', {
   // Specify the other units that are required for this test.
   needs: [
+    'model:custom-inflector-rules',
```

Comment:
Model tests require model itself and it's direct details and masters, is not it?

Reply:
Flexberry generator generates tests like so.
And if we romove 'model:custom-inflector-rules', then tests will be failed, because inflector will transform type names for 'hasMany' relationships and will search for '...-metadatum' types instead of '...-metadata'.